### PR TITLE
feat: provide an ability to ignore invalid SSL certificates while usi…

### PR DESCRIPTION
**Pull request by oleh-momot**
…ng LDE

### CODE REVIEW

File: ilc/server/tailor/server-router.spec.js [hunk 1]
```diff
@@ -444,7 +446,7 @@ describe('server router', () => {
         })).toString('base64'));
 
         const expectedPage =
-            `<fragment id="${registryConfig.apps['apps'].name}" slot="apps" timeout="${registryConfig.apps.apps.ssr.timeout}" src="${expectedAppsUrl.toString()}" primary="true"></fragment>`;
+            `<fragment id="${registryConfig.apps['apps'].name}" slot="apps" timeout="${registryConfig.apps.apps.ssr.timeout}" src="${expectedAppsUrl.toString()}" ignore-invalid-ssl="false" primary="true"></fragment>`;
```
Comment (StyleT) [inline ilc/server/tailor/server-router.spec.js:449]: `ignore-invalid-ssl="false"` counts as true.

**Review by StyleT — CHANGES_REQUESTED** [review]

File: ilc/server/tailor/parse-override-config.spec.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
 import { expect } from 'chai';
 const parseOverrideConfig = require('./parse-override-config');
 
+const assignIgnoreInvalidSsl = (ssr, protocol = 'http:') => protocol === 'https:'
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply ilc/server/tailor/parse-override-config.spec.js:4]: Added

File: ilc/server/tailor/server-router.spec.js (hunk 1 above)
Comment (oleh-momot) [reply ilc/server/tailor/server-router.spec.js]: Fixed

File: ilc/server/tailor/parse-override-config.js [hunk 3]
```diff
@@ -45,6 +45,17 @@ module.exports = (cookie, trustedOrigins) => {
                 const parsedTrustedOrigin = typeof trustedOrigins === 'string' && trustedOrigins.split(',').map(n=>n.trim());
                 sanitizeSpoofedLinks(overrideConfig.apps, parsedTrustedOrigin);
             }
+
+            if (overrideConfig.apps !== undefined) {
+                for (let appName in overrideConfig.apps) {
+                    const ssr = overrideConfig.apps[appName].ssr;
+
+                    if (ssr !== undefined && ssr.src !== undefined && ssr.src.startsWith('https:')) {
+                        overrideConfig.apps[appName].ssr.ignoreInvalidSsl = true;
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply ilc/server/tailor/parse-override-config.js:59]: Provided a comment to explain it

File: ilc/server/tailor/parse-override-config.spec.js [hunk 4]
```diff
@@ -95,6 +99,32 @@ describe('overrideConfig', () => {
         });
     });
 
+    describe('ignoring invalid SSL certificates while SSR', () => {
```
Comment (oleh-momot) [inline ilc/server/tailor/parse-override-config.spec.js:102]: TODO
Add test to check when an app does not have SSR object as a property

**Review by StyleT — APPROVED** [review]

**Pull request merged**